assets/chat/js/messages.js

```diff
@@ -141,6 +141,12 @@ class MessageBuilder {
         return m;
     }
 
+    static whisperoutgoing(message, user, targetoutgoing, timestamp = null){
+        const m = new ChatUserMessage(message, user, timestamp);
+        m.targetoutgoing = targetoutgoing;
```

Comment:
should we add this new property to the class itself (like i.e. `historical`)?

Reply:
Like in line 220, or am I missing something?

Reply:
yeah that works

Reply:
wait how did I miss that 🤦‍♂️